src/types/geohash.h

```diff
@@ -59,6 +59,8 @@ enum GeoDirection {
   GEOHASH_NORT_EAST
 };
 
+enum GeoShapeType { NONE = 0, CIRCULAR, RECTANGULAR };
```

Comment:
You can name enum values using the `kGeoShapeType` prefix: `kGeoShapeTypeNone`, `kGeoShapeTypeCircular`, `kGeoShapeTypeRectangular`. Or use an enum class (for example, `GeoShapeType::kNone` etc.)

Reply:
ack, renaming it